tyr-runner/src/main/java/org/jboss/tyr/model/TyrConfiguration.java

```diff
@@ -17,4 +17,18 @@
     @ConfigProperty(name = "whitelist.enabled", defaultValue = "false")
     boolean whitelistEnabled();
 
+    @ConfigProperty(name = "github.oauth.token")
+    String oauthToken();
+
+    @ConfigProperty(name = "users.dir")
+    Optional<String> whitelistDir();
+
+    @ConfigProperties(prefix = "teamcity")
+    interface TeamCityProperties {
```

Comment:
I personally don't like this design. This is "outside" package private interface so it should be in a separate file. Just make the prefix = "tyr.teamcity"